taxi/views.py

```diff
@@ -1,3 +1,17 @@
 from django.shortcuts import render
```

Comment:
There is no need to add a blank line for imports from the same import group. Please, remove one blank line